tslint.json

```diff
@@ -64,51 +62,23 @@
       "single"
     ],
     "radix": true,
-    "semicolon": [true, "always"],
+    "semicolon": false,
```

Comment:
What about this one? I personally like consistency and force semicolons

Reply:
will change it